test/flask/snaps.spec.ts

```diff
@@ -13,6 +13,27 @@ function getSnapIdByName(testContext: TestContext, snapName: Snaps): string {
 describe("snaps", function () {
   let metamask: dappeteer.Dappeteer;
 
+  async function installPermissionSnap(
```

Comment:
maybe it would be best to create new snap for testing invocation of methods?

Reply:
Not sure if we really need a new snap right now as in the previous test we are just installing it and there is not much interaction with that. So maybe in the future when we will have some logical separation between the test cases and operations with the snap we might require one more. For now, it will add to the repo many new files and not much value.

Reply:
@mpetrunic correct me if I am wrong. Your idea is to create "dialog-snap" in test/flask/dialog-snap which will be similar to permission snap but serve as a testing ground for dialog-based methods. Is it right?

Reply:
Currently, there are a few additional features from snap ([list of methods](https://docs.metamask.io/guide/snaps-rpc-api.html#table-of-contents)) and we want to have a snap for all these features.
Currently is only dialog and inApp notification, but we want to be future proof especially when we know that there will be more features coming.

Reply:
something like that, could be even feature-snap or sample-snap which has the snap method to test each feature